src/server/query.ml

```diff
@@ -28,8 +28,11 @@ let all_queries request =
   |> snd
   |> Formats.from_form_urlencoded
 
-let query request name =
-  List.assoc_opt name (all_queries request)
+let query request name ?(filter_query = false) () =
+  match (List.assoc_opt name (all_queries request), filter_query) with
+  | None, _ -> None
+  | Some v, false -> Some v
+  | Some v, true -> if String.trim v = "" then None else Some v
```

Comment:
It will `trim` multiple occurrences of spaces. That's not what was described in the original issue but if we only check for one empty space, `" "` will give `None` while multiple spaces will result in `Some("  ")`. It looks a bit weird to me but I'm definitely open to changing it if you prefer.